src/backend/app/Providers/AppServiceProvider.php

```diff
@@ -110,5 +111,13 @@ public function register(): void {
             UserCreatedEvent::class,
             UserListener::class
         );
+
+        // App\Listeners namespace
+        // manually register TransactionSuccessfulEvent to listener.
+        // because TransactionSuccessfulEvent is also registered in SparkMeter namespace.
```

Comment:
I'm not sure I fully understand this comment... Because we are registering the listeners in SparkMeter namespace the auto discover doesn't work? Can't we just remove them from SparkMeter instead?

Reply:
SparkMeter binds the `TransactionSuccessfulEvent` event to a SparkMeter-specific Listener, so removing it will break its functionality with the Plugin.